client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1592,8 +1592,8 @@
                                 authoringWorkspace.close();
                             }
                         }, function(err) {
-                            if (err.data._issues['validator exception']) {
```

Comment:
We still need this. In other cases we receive error message in validator exception